<!-- -->
<launch>

	<arg name="detection_range" default="1.3" /> <!-- meter 2.2 -->
	<arg name="threshold_points" default="15" /> <!-- 5 -->
	<arg name="stop_interval" default="15" /> 
	<arg name="detection_height_top" default="2.0" /><!-- 2.0 -->
	<arg name="detection_height_bottom" default="-1.0" />
	<arg name="current_pose_topic" default="ndt" /> 
	<arg name="velocity_limit" default="1.94" />

	<!-- rosrun waypoint_follower velocity_set _detection_range:=0 -->
	<node pkg="waypoint_follower" type="velocity_set" name="velocity_set" output="screen">
		<param name="detection_range" value="$(arg detection_range)" />
		<param name="threshold_points" value="$(arg threshold_points)" />
		<param name="stop_interval" value="$(arg stop_interval)" />
		<param name="detection_height_top" value="$(arg detection_height_top)" />
		<param name="detection_height_bottom" value="$(arg detection_height_bottom)" />
		<param name="current_pose_topic" value="$(arg current_pose_topic)" />
		<param name="velocity_limit" value="$(arg velocity_limit)" />
	</node>

</launch>
